project/Build.scala

```diff
@@ -83,9 +83,9 @@ object DottyJSPlugin extends AutoPlugin {
 object Build {
   import ScaladocConfigs._
 
-  val referenceVersion = "3.3.1"
+  val referenceVersion = "3.4.0-RC1"
```

Comment:
We should either wait to have `3.4.0-RC2` that has the correct minor TASTy version or keep using `3.3.1`

Reply:
shouldn't this be `3.4.0`? why would we start `3.4.1` before `3.4.0` was released?